.github/workflows/ci.yml

```diff
@@ -35,20 +35,31 @@ jobs:
     - name: Setup
       run: |
         sudo apt-get update && sudo apt-get install wget unzip -y
-        rm -rf carla_msgs && wget https://github.com/carla-simulator/ros-carla-msgs/archive/master.zip
-        unzip master.zip -d carla_msgs && rm master.zip
+        #TODO: replace ros2 with master
+        rm -rf carla_msgs && wget https://github.com/carla-simulator/ros-carla-msgs/archive/ros2.zip
+        unzip ros2.zip -d carla_msgs && rm ros2.zip
         packaging/install_dependencies.sh
     - name: Init Workspace
+      if: matrix.ros-version != 'foxy'
```

Comment:
To avoid the doubled check for ROS 1, we could put the code of the init workspace step into the ROS 1 build step